packages/flutter/lib/src/material/bottom_navigation_bar.dart

```diff
@@ -51,6 +51,9 @@ enum BottomNavigationBarType {
 /// [BottomNavigationBarType.fixed] when there are less than four items, and
 /// [BottomNavigationBarType.shifting] otherwise.
 ///
+/// The length of [items] must be at least two and each item's icon and title
```

Comment:
I guess title/label should be more convenient.

Reply:
Done Thanks @yash1200